spec/spec_helper.rb

```diff
@@ -10,6 +10,10 @@
 SimpleCov.start "rails" do
   add_filter "/mcb/"
 end
+SimpleCov.start do
```

Comment:
Why is this here?

Reply:
Specs were being included as part of the coverage